src/components/SignUp/OffstageRoles.tsx

```diff
@@ -1,22 +1,263 @@
 import React from 'react';
-import Container from 'react-bootstrap/Container';
+import { fonts } from '../../theme/styleVars';
+import styled from 'styled-components';
+import GenericAccordion from '../../genericComponents/Accordion';
+import Checkbox from '../../genericComponents/Checkbox';
+import yellow_blob_1 from '../../images/yellow_blob_1.svg';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
+import Image from 'react-bootstrap/Image';
+import Container from 'react-bootstrap/Container';
```

Comment:
[nit] Make sure to re-order your inputs so that they are like:
1. React
2. styled-components
3. Outside imports (like the bootstrap imports)
4. Local component imports
5. images or other local media assets or stylesheets